src/sass/static/home.scss

```diff
@@ -184,3 +184,135 @@
         }
     }
 }
+
+#redirect_popup_container {
```

Comment:
why not use our existing lightbox pop up?

Reply:
I can use .lightbox in components.scss, but it has some styles for first element 
`> div {
        margin: auto;
        max-width: 44em;
        max-height: 80vh;
        ....`
so I have to overwrite most of them. 
But if you prefer to use it, I'll change my DOM and styles to fit it.

Reply:
I mean can we use the existing dialog or popup functions to render this or it needs something completely different?

Reply:
I've check them, I don’t think so. They need some changes to show custom HTML.
Dialog is generating a popup with confirm/cancel button and there is no method to add custom HTML. Our popup design has title and checkbox and also contains a close button icon. 
showPopup is generating a form(we don’t need form), and doesn’t have method to add custom HTML like Dialog.

Reply:
I've checked them, I don’t think so. They need some changes. 
Dialog is generating a popup with default confirm/cancel button and there is no method to add custom HTML. Our popup design has title and checkbox and also contains a close button icon. 
showPopup can show custom HTML but it is a form validator popup, we don’t need form.

showPopup: src/javascript/app/common/attach_dom/popup.js 
Dialog: src/javascript/app/common/attach_dom/dialog.js

If there are other functions to show popup please let me know.